tbx/core/blocks.py

```diff
@@ -329,6 +329,30 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/work_chooser_block.html"
 
 
+class TabbedParagraphBlock(blocks.StructBlock):
+    title = blocks.CharBlock(max_length=255)
+    intro = blocks.TextBlock(label="Introduction")
+    tabbed_paragraph_sections = blocks.ListBlock(
+        blocks.StructBlock(
+            [
+                ("name", blocks.CharBlock()),
+                ("summary", blocks.TextBlock()),
+                ("text", blocks.RichTextBlock()),
+                ("button_text", blocks.CharBlock(required=False)),
+                ("button_link", blocks.PageChooserBlock(required=False)),
+            ],
+            help_text="Add a tabbed paragraph, with a name, summary, text and an optional page link & button text",
+            icon="breadcrumb-expand",
+        ),
+        min_num=2,
+        help_text="Add at least two tabbed paragraph sections",
+    )
+
+    class Meta:
+        icon = "list-ul"
+        template = "patterns/molecules/streamfield/blocks/tabbed_paragraph_block.html"
+
```

Comment:
Comment: I guess its not really a necessity, but would be good to have a check here to ensure that either both or neither of `button_text` and `button_link` are added.

Reply:
I was having some difficulty getting the error to raise on the field (or at least the section) itself rather than on the whole block. I tried enumerating thought the tabbed_paragraph_sections but I can't find a way to get this error to raise in a more targeted way. Is there a way you can see to do this? 
![image](https://github.com/torchbox/torchbox.com/assets/68611528/2ee84613-e90b-4c3a-8507-0f31994a5a02)

Reply:
I don't think there is a way to display it on the specific block, but this should be good enough :)